style-guide.md

```diff
@@ -74,7 +74,7 @@ React elements are [immutable](https://en.wikipedia.org/wiki/Immutable_object).
 ✅ OK:
 
 ```md
-Los elementos de React son [inmutables](https://es.wikipedia.org/wiki/Objeto_inmutable).
+Reactin elementit ovat muuttumattomia.
```

Comment:
Should the original link be here if there is not a translated one?

Reply:
Yeah, perhaps.